bandit/core/config.py

```diff
@@ -15,10 +15,7 @@
     except ImportError:
         tomllib = None
 
-from bandit.core import constants
-from bandit.core import extension_loader
-from bandit.core import utils
-
+from bandit.core import constants, extension_loader, utils
```

Comment:
Bandit follows PEP8 which states "Imports should usually be on separate lines"

Reply:
interesting, looks like isort doesn't always follow pep8. fixed.